assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -364,6 +371,11 @@ const SearchFunnelWidget = ( {
 				/>
 			) }
 
+			{ ! isAnalyticsActive &&
+				! isAnalyticsConnected &&
+				zeroDataStatesEnabled &&
+				BREAKPOINT_SMALL === breakpoint && <ActivateAnalyticsCTA /> }
```

Comment:
@hussain-t Let's wrap `ActivateAnalyticsCTA` within `Grid -> Row -> Cell` so that we can match the designs in Figma. If you look at the screenshot in the "Feature description" of the ticket, you will notice that there is some padding around the component.
Current:
<img width="337" alt="image" src="https://user-images.githubusercontent.com/31818/157485213-2a3e0dc2-0f2c-45e6-b874-a136bf14f1dd.png">

Expected:
<img width="448" alt="image" src="https://user-images.githubusercontent.com/31818/157485359-9c43b7c4-0a1a-4ff3-8280-9c082506f6ea.png">
```suggestion
<Grid>
						<Row>
							<Cell>
								<ActivateAnalyticsCTA />
							</Cell>
						</Row>
					</Grid>
```

Reply:
Thanks, @asvinb that's a good catch 👍